climada/entity/exposures/base.py

```diff
@@ -118,15 +118,20 @@ class Exposures():
     @property
     def crs(self):
         """Coordinate Reference System, refers to the crs attribute of the inherent GeoDataFrame"""
-        try:
-            # it is very well possible that gdf.crs is None while the geometry columns has a crs
-            return self.gdf.crs if self.gdf.crs else \
-                   self.gdf.geometry.crs if 'geometry' in self.gdf and self.gdf.geometry.crs else \
-                   self.meta.get('crs')
-        except AttributeError:
-            return self.meta.get('crs')
-
-    def __init__(self, *args, **kwargs):
+        if hasattr(self.gdf, "crs") and self.gdf.crs:
+            return self.gdf.crs
+        # it is very well possible that gdf.crs is None while the geometry columns has a crs
+        if "geometry" in self.gdf and hasattr(self.gdf.geometry, "crs") and self.gdf.geometry.crs:
+            return self.gdf.geometry.crs
+        return self.meta.get('crs')
+
+    def __init__(self, *args,
+                 meta=None,
+                 tag=None,
+                 ref_year=DEF_REF_YEAR,
+                 value_unit=DEF_VALUE_UNIT,
+                 crs=None,
```

Comment:
IMHO, having these as explicit keyword arguments improves readability of both the docs and the code a lot.

Reply:
👍 Totally!